src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -388,7 +391,9 @@
             :cipher-suites           (get-cipher-suites-config config)
             :protocols               (get-ssl-protocols-config config)
             :client-auth             (get-client-auth! config)
-            :ssl-crl-path            (get-ssl-crl-path! config)})))
+            :ssl-crl-path            (get-ssl-crl-path! config)
+            :allow-renegotiation     (or (:allow-renegotiation config) 
```

Comment:
If this is set to `false` explicitly in the config and we change the default to `true` users won't be able to override this, right? I think this pattern works above because the config values are expected to *not* be booleans, since we're expecting booleans here we might want to do something more like:

```clj
(get config :allow-renegotiation default-allow-renegotiation)
```

Reply:
Just noticed this comment.. let me ponder upon it and get back.

Reply:
I agree your proposal promotes genericity though I would not think (or have liked) allowing renegotiation by default as possible usage except in some completely security agnostic situations etc. In any case I have done the needful.   thanks